include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -435,71 +435,28 @@ struct __sycl_usm_alloc
     }
 };
 
-//-----------------------------------------------------------------------
```

Comment:
Can we avoid moving of this code?
I think it is a little bit difficult to understands your changes when this code were moved.

Reply:
I agree that this convolutes the review. However, the intention was to move them into the `__internal` namespace to align better with other helpers.

Reply:
ok